unlock-app/src/components/interface/checkout/main/Confirm/ConfirmCard.tsx

```diff
@@ -193,13 +194,23 @@ export function ConfirmCard({ checkoutService, onConfirmed, onError }: Props) {
   // show gas cost only when custom credit card price is present
   const gasCosts = creditCardPrice ? undefined : totalPricing?.gasCost
 
+  useEffect(() => {
```

Comment:
Since this code is duplicated, what about introducing a `useReferrers` hook (inside of the `hooks` folder) that we can re-use more easily?